sdk_lib/enter_chroot.sh

```diff
@@ -185,9 +185,9 @@ generate_locales() {
     # with long multibyte strings.  Newer setups have this fixed,
     # but locale-gen doesn't need to be run in any locale in the
     # first place, so just go with C to keep it fast.
-    PATH="/usr/sbin:/usr/bin:/sbin:/bin" LC_ALL=C \
-      chroot "${FLAGS_chroot}" locale-gen -q -u \
-      -G "$(printf '%s\n' "${gen_locales[@]}")"
+    chroot "${FLAGS_chroot}" /usr/bin/env \
```

Comment:
The stuff we execute under the chroot shouldn't need any changes WRT the host environment, just changing the shebang lines of these scripts which run on the host to include the /usr/bin/env should be sufficient, no?